source/backend/api/Areas/Documents/DocumentController.cs

```diff
@@ -142,6 +142,19 @@ public async Task<IActionResult> GetDocumentStorageTypeMetadata(long mayanDocume
             return new JsonResult(result);
         }
 
+        /// <summary>
+        /// Retrieves the document detail for a document type.
+        /// </summary>
```

Comment:
nit: this method seems to be getting doc details by its ID but comment says it's for a doc type